.github/workflows/pr.yml

```diff
@@ -9,17 +9,19 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
-      - name: Use Node.js 18.8.x
+      - name: Use Node.js LTS versions
         uses: actions/setup-node@v3
         with:
           # use node 18.8.x until Agoric/agoric-sdk#8636
-          node-version: "18.8.x"
+          node-version: ['18.x', '20.x']
```

Comment:
@dckc Have you tried changing this to just a string instead of an array?
@frazarshad suggests:
"i think instead we can use [matrix testing](https://github.com/actions/setup-node?tab=readme-ov-file#matrix-testing) if we want to test on multiple node versions"

Reply:
This was an attempt to do matrix testing. We should be testing on 18 and 20. Ah... it look like [patrick's comment](#issuecomment-2104906074) is saying roughly the same thing. I can try that...